material/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 28
```

Comment:
I think this is probably safe to update across the project? Not very important, though.

Reply:
Yeah thought we migth as well keep it the same right now to avoid any potential problems.